sample/src/enhancedphotography.js

```diff
@@ -419,9 +407,9 @@ function main() {
     }
     XDMUtils.saveXDM(savePhoto).then(
         function(blob) {
-          xwalk.experimental.native_file_system.requestNativeFileSystem('pictures',
+          RSUtils.nativeFS.requestNativeFileSystem('pictures',
```

Comment:
I would suggest not to wrap Crosswalk API. I am fine to use alias for short name. But in a explicit way. One important purpose of our sample is to showcase the API usage.